### DISCUSSION TITLE
cn/2024/11/21/dreams/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#188] [Announcements]
# cn/2024/11/21/dreams/

https://hongtaoh.com/cn/2024/11/21/dreams/

**Comment by Liam-Zhong** [comment]
想问下博主怎么看待一些诸如「周公子，北极鲶鱼」之类的人呢？一个人的收入和他对这个社会的贡献大体上是成正比的，我也很愿意相信，但是落到现实，那么多加班给老板换宝马的事情，又该如何理解呢？

**Reply by hongtaoh** [reply]
>博主怎么看待一些诸如「周公子，北极鲶鱼」之类的人呢？

我只能从宗教的角度来解释。一个人不仅生下来在财富上不平等，在别的方面也不平等，比如身高、长相、智商、家庭教育、性格。从佛教因果的角度讲，这些人生下来就有钱，是因为之前行善的结果。

>但是落到现实，那么多加班给老板换宝马的事情，又该如何理解呢

这个我觉得用现实的角度就可以解释。老板之所以是老板，是因为他承担了最初创业失败的风险，成立了公司，找了人。员工给老板打工，自然是在给老板换宝马。至于加班，这个是整个社会的问题了。

**Comment by wangyunzi** [comment]
理想不是笑话，胡思乱想才是🤣

**Comment by Jacob-jinxy** [comment]
「百万AI」sounds interesting! 有什么能volunteer能做的吗

**Reply by hongtaoh** [reply]
感谢关注！一个临时的想法，不知道能持续多久。目前是我先要毕业。github 你常用的我关注你就好。不常用的话可以留一个联系方式。
